plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -51,15 +51,17 @@ global proc int mayaUsdTranslatorImport (string $parent,
 
         columnLayout -adj true -rs 5 mayaUsdTranslator_OptsCol;
 
-        rowLayout -numberOfColumns 3 -cw 1 $cw1 -cat 1 "right" 0;
-            text -label "Scope and Variants: " -al "right" -ann "Select a USD file and click Hierarchy View to build the scope of your import and switch variants" ;
-            textField -ed false -nbg true -text "Please select a file to enable this option" -w $cw2 mayaUsdTranslator_SelectFileField;
-            button -label "Hierarchy View" -c "string $usdFile = currentFileDialogSelection(); usdImportDialog $usdFile;" mayaUsdTranslator_ViewHierBtn;
-
-            // Hide the button by default.
-            button -e -visible false mayaUsdTranslator_ViewHierBtn;
-        setParent ..;
-
+        if (`exists usdImportDialog`)
```

Comment:
The command usdImportDialog will only be available when compiling with the optional Qt features enabled (by using --qt-location). So when not available we need to protect the import translator UI by not showing the "Hierarchy View" button.